Gravity/Gravity/Base/BaseExtensionMethods.cs

```diff
@@ -79,7 +79,21 @@ public static String GetEnumDescriptionAttributeValue(this Enum enumeration)
 			return body.Member.Name;
 		}
 
-		public static void SetValueByPropertyName(this object input, string propertyName, object value)
+	    public static TAttribute GetCustomAttibute<TAttribute>(this object obj, string propertyName) where TAttribute : Attribute
+	    {
+	        TAttribute fieldAttribute = obj.GetType().GetPublicProperties()
+	            .FirstOrDefault(property => property.Name.Equals(propertyName, StringComparison.OrdinalIgnoreCase))?
+	            .GetCustomAttribute<TAttribute>();
+	        return fieldAttribute;
+	    }
+
+	    public static TAttribute GetCustomAttibutes<TAttribute>(this object obj) where TAttribute : Attribute
+	    {
+	        TAttribute fieldAttribute = obj.GetType().GetCustomAttributes<TAttribute>().FirstOrDefault();
```

Comment:
Why not just use GetCustomAttribute (and rename this method accordingly?)

Reply:
I added this for consistency.  It allows testObject.GetCusttomAttributes like we access the RelativityObjectFieldAttribute.  If not we have to testObject.GetType().GetCustomAttribute.

Reply:
Alright. This is misspelled, though - it should also be `GetCustomAttribute`.
After that, I think we're good to go!

Reply:
Woohoo...I had to rename the function to  GetObjectLevelCustomAttribute because GetCustomAttribute conflicted with default extension method.  I believe we are all good!  If there is any way we can get this merged by early afternoon that would be awesome.  I think @ppande1020 is working on some more of the CI stuff she would like to incorporate some of these changes with.